net-mgmt/zabbix-agent/src/opnsense/scripts/OPNsense/ZabbixAgent/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+AGENT_DIRS="/var/run/zabbix /var/log/zabbix /usr/local/etc/zabbix_agentd.d"
```

Comment:
Well, in theory the package creates `%%ETCDIR%%/zabbix_agentd.conf.d`, but it doesn't work for me. I've changed the code to use this (FreeBSD) default directory instead.